src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -219,6 +257,13 @@ def prepare_logs_argument(c):
             c.argument('env', env_type)
             c.argument('disable_probe', arg_type=get_three_state_flag(), help='If true, disable the liveness and readiness probe.')
 
+    for scope in ['update', 'deployment create', 'deploy']:
+        with self.argument_context('spring-cloud app {}'.format(scope)) as c:
+            c.argument('config_file_patterns', type=str,
```

Comment:
`type=str` is the default setting, so we don't need an explicit declaration

Reply:
removed